backend/README.md

```diff
@@ -13,19 +13,61 @@ Furthermore, the `Snapshot` struct contains the following methods:
 - `generate_solvency_verifier` -> write the Solidity Verifier contract (for the `SolvencyProof`) to a file
 - `generate_proof_of_solvency` -> generate the `SolvencyProof` for the current snapshot to be verified on-chain
 - `generate_inclusion_proof` -> generate the `MstInclusionProof` for a specific user for the current snapshot to be verified off-chain
-- `get_account_onwership_proof` -> generate the `AccountOwnership` for a specific user for the current snapshot to be verified off-chain
+- `get_proof_of_account_ownership` -> generate the `AccountOwnership` for a specific user for the current snapshot to be verified off-chain
 
 ## Prerequisites
 
-In order to initialize the Snapshot, you need to download the Powers of Tau files. These are the trusted setup parameters needed to build the zk circuits. You can find such files at https://github.com/han0110/halo2-kzg-srs, download it
+The `ptau` file, containing the Powers of Tau trusted setup parameters needed to build the zk circuits, is already included. However, if you wish to test or run the code with a higher number of entries, you may choose to download a different `ptau` file.
+
+You can find the necessary files at https://github.com/han0110/halo2-kzg-srs. To download a specific file, you can use:
 
 ```
 wget https://trusted-setup-halo2kzg.s3.eu-central-1.amazonaws.com/hermez-raw-11
 ```
 
-and pass the path to the file to the `Snapshot::new` method.
+After downloading, pass the path to the desired file to the `Snapshot::new` method. If you are using the included `ptau` file, no additional steps are necessary.
+
+## Important Notice for Proof Generation
+
+### Proof of Solvency
+
+As of the current implementation, the `generate_proof_of_solvency` method does not directly fetch data about the balances of the wallets of the CEX. Instead, you can use the `fetch_asset_sums` function to retrieve balance information from the blockchain. Here's an example of how you might utilize it:
+
+```Rust
+let asset_sums = fetch_asset_sums(client, token_contracts, exchange_addresses).await?;
+```
+
+Please note that the first element in the `asset_sums` array represents the ETH balance.
+
+Alternatively, you can create your own custom fetcher to retrieve the balances.
+
+### Proof of Ownership
```

Comment:
In this section we explain how to generate signatures in order to generate a `proof_of_account_ownership`. But actually the signatures needs to be provided before, when initializing the `Snapshot` within a `csv` file.

So maybe it's better to remove the `proof of ownership` section and add a section to the readme that explains what are the input needed to initalize the `Snapshot`. When mentioning the `csv` file containing the signatures, you can mention that we have a method that helps generating a signature starting from a message and a private key. In any case the signatures are to be added to a `csv` file  to be processed into  a proof of account ownership

Reply:
Removed at [70ad9c9](https://github.com/summa-dev/summa-solvency/pull/116/commits/70ad9c97f31c3573941a23cb0ff7a36952f95392)